policy/DecamMapper.yaml

```diff
@@ -255,12 +253,8 @@ datasets:
     template: deepDiff/v%(visit)d/kernelSrc-%(ccdnum)02d.fits
   dcrCoadd_measMatch:
     template: dcrCoadd-results/%(filter)s%(subfilter)s/%(tract)d/%(patch)s/srcMatch-%(filter)s%(subfilter)s-%(tract)d-%(patch)s.fits
-  deepCoadd_measMatchFull:
-    template: deepCoadd-results/%(filter)s/%(tract)d/%(patch)s/srcMatchFull-%(filter)s-%(tract)d-%(patch)s.fits
   dcrCoadd_forced_src:
     template: dcrCoadd-results/%(filter)s/%(tract)d/%(patch)s/forced-%(filter)s-%(tract)d-%(patch)s.fits
-  dcrCoadd_forced_config:
-    template: config/dcrCoadd_forced.py
```

Comment:
`dcrCoadd_measMatch`, `dcrCoadd_forced_src`, and `dcrCoadd_forced_metadata` should also be deleted.

Reply:
`dcrCoadd_measMatchFull`, too?  What's there for that now is not obviously wrong, but it seems unnecessarily different from what's in `obs_base`:

obs_decam:
```
dcrCoadd-results/%(filter)s%(subfilter)s/%(tract)d/%(patch)s/srcMatchFull-%(filter)s%(subfilter)s-%(tract)d-%(patch)s.fits
```

obs_base:
```
dcrCoadd-results/%(filter)s%(subfilter)dof%(numSubfilters)d/%(tract)d/%(patch)s/srcMatchFull-%(filter)s%(subfilter)dof%(numSubfilters)d-%(tract)d-%(patch)s.fits
```

Reply:
Yes, `dcrCoadd_measMatchFull` should go too. Sorry I missed it!